aosp_diff/celadon_ivi/frameworks/native/0003-Fix-the-acrn-sriov-vulkan-screen-blur-issue.patch

```diff
@@ -0,0 +1,30 @@
+From 8116ea897a28cbe030f605ec3cc14128a5d0d957 Mon Sep 17 00:00:00 2001
+From: HeYue <yue.he@intel.com>
+Date: Fri, 9 Jun 2023 10:00:38 +0800
+Subject: [PATCH] Fix the acrn sriov vulkan screen blur issue
+
+Set the special bit to indicate the framebuffer.
+
+Tracked-On: OAM-110165
+Signed-off-by: HeYue <yue.he@intel.com>
+---
+ services/surfaceflinger/DisplayHardware/FramebufferSurface.cpp | 3 ++-
+ 1 file changed, 2 insertions(+), 1 deletion(-)
+
+diff --git a/services/surfaceflinger/DisplayHardware/FramebufferSurface.cpp b/services/surfaceflinger/DisplayHardware/FramebufferSurface.cpp
+index 8d685cfbdd7e..c9b41b361687 100644
+--- a/services/surfaceflinger/DisplayHardware/FramebufferSurface.cpp
++++ b/services/surfaceflinger/DisplayHardware/FramebufferSurface.cpp
+@@ -75,7 +75,8 @@ FramebufferSurface::FramebufferSurface(HWComposer& hwc, PhysicalDisplayId displa
+     mConsumer->setConsumerName(mName);
+     mConsumer->setConsumerUsageBits(GRALLOC_USAGE_HW_FB |
+                                        GRALLOC_USAGE_HW_RENDER |
+-                                       GRALLOC_USAGE_HW_COMPOSER);
++                                       GRALLOC_USAGE_HW_COMPOSER |
++                                       0x1000000U);
```

Comment:
Use a macro instead of direct value to indicate for what purpose?

Reply:
The minigbm use this flag to indicate: http://inlubt0232.iind.intel.com:8080/source/xref/CELADON/hardware/interfaces/graphics/common/aidl/aidl_api/android.hardware.graphics.common/1/android/hardware/graphics/common/BufferUsage.aidl?r=7e267458#44
it is 0x1000000U, while in the http://inlubt0232.iind.intel.com:8080/source/xref/CELADON/external/mesa3d/include/android_stub/hardware/gralloc.h?r=932f51d5#96 there is no macro related.
so i simply hard code the flag.
Also Fei said we could try allocate buffer only use i915 while in SRIOV mode, i tried, seems scanout buffer allocate has issue. so the display is black but screen copy works. I need further check.